lychee.example.toml

```diff
@@ -1,59 +1,97 @@
-###
-### Display
-###
+#############################  Display  #############################
+
 # Verbose program output
 verbose = false
 
-# Show progress
-progress = false
+# TODO
+no_progress = false
+
+# TODO
+output = "report.md"
+
+#############################  Cache  ###############################
+
+# TODO
+cache = true
 
+# TODO
+# Error: invalid type: string "2d", expected struct Duration for key `max_cache_age`
```

Comment:
Ouch. Yeah that's something we should fix. I think `max_cache_age` should be `Into<Duration>` instead of `Duration`. And maybe we need an `impl From<String> for Duration` to handle that case or use`human_time`. I can look into this, but for now it's fine to keep it in there.